apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.scss

```diff
@@ -313,3 +321,62 @@ button.btn.btn-filter {
 .group-not-found {
     text-transform: none;
 }
+.tdmobile-head{
+    display:none;
+}
+
+
+@media (max-width:767px){
+    .btn-pink{
+        display:none;
+    }
+    .advance-icon{
+        display:none;
+    }
+    .head-report{
+        margin-right:0 !important;
+        font-size: 16px;
+    }
+    .btn-link{
+        margin-right:0 !important;
+        margin-top:0 !important;
+        span{
+            display:none;
+        }
+    }
+    .inventoryStockTable{
```

Comment:
if you added new CSS then please make sure code should be according to guideline